src/client/eppo-client.ts

```diff
@@ -212,17 +211,22 @@ export default class EppoClient implements IEppoClient {
       );
       return;
     }
-
-    // todo: consider injecting the IHttpClient interface
-    const httpClient = new FetchHttpClient(
-      this.configurationRequestParameters.baseUrl || DEFAULT_BASE_URL,
-      {
-        apiKey: this.configurationRequestParameters.apiKey,
-        sdkName: this.configurationRequestParameters.sdkName,
-        sdkVersion: this.configurationRequestParameters.sdkVersion,
-      },
-      this.configurationRequestParameters.requestTimeoutMs || DEFAULT_REQUEST_TIMEOUT_MS,
-    );
+    const {
+      apiKey,
+      sdkName,
+      sdkVersion,
+      baseUrl = DEFAULT_BASE_URL,
+      requestTimeoutMs = DEFAULT_REQUEST_TIMEOUT_MS,
+      numInitialRequestRetries = DEFAULT_INITIAL_CONFIG_REQUEST_RETRIES,
+      numPollRequestRetries = DEFAULT_POLL_CONFIG_REQUEST_RETRIES,
+      pollAfterSuccessfulInitialization = false,
+      pollAfterFailedInitialization = false,
+      throwOnFailedInitialization = false,
+      skipInitialPoll = false,
+    } = this.configurationRequestParameters;
```

Comment:
this is a simple/straightforward cleanup for readability. no functionality was changed other than introducing `ApiEndpoints` below